packages/store/tests/utils/utils.spec.ts

```diff
@@ -60,4 +62,24 @@ describe('utils', () => {
     expect(ObjectUtils.merge(null as any, null as any)).toEqual({});
     expect(ObjectUtils.merge(new A(), { id: 0 })).toEqual({ value: 'hello world', id: 0 });
   });
+
+  describe('propGetter', () => {
+    const defaultConfig: NgxsConfig = new NgxsConfig();
+    const target: PlainObject = { a: { b: { c: 100 } } };
```

Comment:
These lines should rather be within the tests. It makes the test more readable and independent.